src/Controller/CRUDController.php

```diff
@@ -804,11 +751,10 @@ public function historyViewRevisionAction($id = null, $revision = null) // NEXT_
      *
      * @return Response
      */
-    public function historyCompareRevisionsAction($id = null, $base_revision = null, $compare_revision = null) // NEXT_MAJOR: Remove the unused $id parameter
+    public function historyCompareRevisionsAction(Request $request, $base_revision = null, $compare_revision = null)
```

Comment:
Could we leverage these changes to use camel case in argument names? Is required to keep them with snake case?

Reply:
I'll change them